sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkExecuteStatementOperation.scala

```diff
@@ -282,3 +272,18 @@ private[hive] class SparkExecuteStatementOperation(
     }
   }
 }
+
+object SparkExecuteStatementOperation extends Logging {
+  def getTableSchema(result: DataFrame): TableSchema = {
```

Comment:
Do you mean using `result.queryExecution.analyzed.schema` instead of `result.queryExecution.analyzed.output`? I see. I'll update the scope of refactored function and testcase. Although it changes the previous logic, that will make the test suite the simplest. Thank you.